pkg/cmd/chat/user/user.go

```diff
@@ -134,6 +136,61 @@ func upsertCmd() *cobra.Command {
 	return cmd
 }
 
+func updatePartialCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-user-partial --user [user-id] --set [key-value-pairs] --unset [property-names]",
+		Short: "Partially update a user",
+		Long: heredoc.Doc(`
+			Updates an existing user. The 'set' property is a comma separated list of key value pairs.
+			The 'unset' property is a comma separated list of property names.
+		`),
+		Example: heredoc.Doc(`
+			# Set a user's role to 'admin' and set 'age' to 21. At the same time, remove 'haircolor' and 'height'.
+			$ stream-cli chat update-user-partial --user-id my-user-1 --set role=admin,age=21 --unset haircolor,height
+
+			Check the Go SDK's 'User' struct for the properties that you can use here.
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			userId, _ := cmd.Flags().GetString("user-id")
+			set, _ := cmd.Flags().GetStringToString("set")
```

Comment:
let's get a json string, otherwise no way set a custom other than string

Reply:
Good point. I copied this from the update-channel-partial command, so I'll change that one too.